charts/splunk-connect-for-snmp/values.yaml

```diff
@@ -2,36 +2,59 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-replicaCount: 1
+################################################################################
+# SC4SNMP image settings
+################################################################################
 
 image:
+  # The registry and name of the SC4SNMP image to pull
   repository: ghcr.io/splunk/splunk-connect-for-snmp/container
+  # The policy that specifies when the user wants the SC4SNMP images to be pulled
   pullPolicy: Always
   # Overrides the image tag whose default is the chart appVersion.
   tag: ""
 
+# Secrets to attach to the respective serviceaccount to pull docker images
 imagePullSecrets: []
 
-splunk:
-  enabled: true
 
-inventory:
-  nameOverride: ""
-  fullnameOverride: ""
+################################################################################
+# Splunk Data-to-Everything Platform configuration
```

Comment:
I would avoid this branding. Look at SCK Otel chart's issues/636

Reply:
Done